omniauth-gplus.gemspec

```diff
@@ -26,7 +26,8 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'kramdown'
   s.add_development_dependency 'pry'
   s.add_development_dependency 'coveralls'
-  s.add_development_dependency 'rubocop'
+  s.add_development_dependency 'rubocop', '0.24.1'
```

Comment:
@samdunne Yep, good idea. Will do this shortly. Could you reduce check list in travis?